src/Http/Http.Abstractions/src/Microsoft.AspNetCore.Http.Abstractions.csproj

```diff
@@ -20,6 +20,7 @@ Microsoft.AspNetCore.Http.HttpResponse</Description>
   </ItemGroup>
 
   <ItemGroup>
+    <Reference Include="Microsoft.AspNetCore.Authorization" />
```

Comment:
This makes me go hmmm. Is this because of the attributes? What in this project is using them?

Reply:
Yeah we moved IAuthorizeData and IAllowAnonymous back to Authorization

Reply:
What depends on those types in `Http.Abstractions`? Was it like this before? 

@davidfowl @Tratcher do we want this dependency here?

Reply:
I think it comes from routing: https://github.com/aspnet/AspNetCore/blob/23efa15112ae2e7c9bf26e75f7173463b9d9a056/src/Http/Routing/src/EndpointMiddleware.cs#L45

Reply:
I'd want to see the graph. Dependencies don't matter as much as the used to, but we still need to avoid a rats nest.

Reply:
AuthZ doesn't bring much, and nothing new, it just adds Options/Logging.Abstractions which are already referenced

Reply:
That would dictate routing referencing auth though, not http.abstractions

Reply:
I'll try moving the ref to routing

Reply:
Much better